### DISCUSSION TITLE
Add title to tsai plots

### DISCUSSION THREAD

**Discussion by eyalbetzalel** [timeseriesAI/tsai#132] [Q&A]
Hi!

Is there a way to add custom title / axes names to the plots functions such as:

learn.show_results()
learn.show_probas()

etc..

Thanks! :)

**Comment by williamsdoug** [comment]
Hi @eyalbetzalel.  While I haven't tested it, from the code it appears that ```learn.show_results(title='your title')```  should just work.  TSAI calls the Fastai function [```get_grid()```](https://github.com/fastai/fastai/blob/ab154927696338741e59e0ffc4774777c4a9781c/fastai/vision/data.py) that does the actual plot set-up. 

In terms of axes, take a look at ```get_grid()``` and see if it can be configured to meet your needs.  Extra keywords are passed along from tsai ```show_results/show_probas()```  to ```get_grid()```.  You might also try checking the [fastai forum](https://forums.fast.ai/) since this issue is non-specific to tsai.  Worst-case, you could bypass ```get_grid()```setup  by using the ```ctxs``` parameter
